infrastructure/staging/main.tf

```diff
@@ -0,0 +1,126 @@
+terraform {
+  required_version = ">= 0.13"
```

Comment:
We talked about this in Slack, but it would be helpful to add this to the `.tool-versions` file.